scripts/install_platform_with_component_manager.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+CURRENT_DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)
+source "${CURRENT_DIR}/formats.sh"
+
+if [ -z "${SC_MACHINE_PATH}" ];
+then
+  source "${CURRENT_DIR}/set_vars.sh"
+fi
+
+stage "Install ostis-web-platform"
+
+"${PLATFORM_PATH}/scripts/install_sc_machine_submodule.sh"
+"${PLATFORM_PATH}/scripts/install_sc_machine_dependencies.sh" --dev
+"${PLATFORM_PATH}/scripts/install_sc_component_manager_dependencies.sh"
+"${PLATFORM_PATH}/scripts/build_sc_machine.sh" -cm
```

Comment:
When `install_sc_component_manager_dependencies.sh` will be moved into sc-machine you can do that platform installs with comonent manager by `install_platform.sh` with argument `--cm`

Reply:
Not fixed

Reply:
Not necessary